docs/UserGuide.adoc

```diff
@@ -53,7 +53,7 @@ Without further ado, let's head over to <<Quick Start>> to get started!
 To get Know-It-All up and running:
 
 .  Ensure that you have Java version `9` or later installed in your Computer.
-.  Download the latest `know-it-all.jar` link:{repoURL}/releases[here].
+.  Download the latest `knowitall.jar` link:{repoURL}/releases[here].
```

Comment:
This link is not yet updated!
```suggestion
.  Download the latest `knowitall.jar` https://github.com/cs2103-ay1819s2-w10-4/main/releases[here].
```

Reply:
Actually, I realise that repoURL is defined at the top of the doc to be 'https://github.com/cs2103-ay1819s2-w10-4/main' already! But i'm not sure if tagging this portion this the PPP will allow this link to work there! Should I just change it?

Reply:
Yes I just realised it actually does work HAHA. Previously, the one for Model.java wasn't working, I am now just seeing that it was because repoURL was concatenated with the wrong path. Either ways both will work! Thanks for pointing it out!